TECHMANIA/Assets/Scripts/Components/Main Menu/SelectSkinPanel.cs

```diff
@@ -30,11 +30,25 @@ public class SelectSkinPanel : MonoBehaviour
     private List<GameObject> vfxInstances;
 
     private void InitializeDropdown(TMP_Dropdown dropdown,
-        string skinFolder, string currentSkinName)
+        string skinFolder, string skinSaFolder, string currentSkinName)
     {
         dropdown.options.Clear();
         int value = 0, index = 0;
         bool foundOption = false;
+        foreach (string folder in
+            Directory.EnumerateDirectories(skinSaFolder))
```

Comment:
This will throw DirectoryNotFoundException if the streaming assets folder doesn't exist.